docs_user/modules/proc_adopting-the-loadbalancer-service.adoc

```diff
@@ -2,42 +2,263 @@
++
+----
+$ cat >> octavia-nad.yaml << EOF_CAT
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  labels:
+    osp/net: octavia
+  name: octavia
+  namespace: openstack
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
+      "name": "octavia",
+      "type": "bridge",
+      "bridge": "octbr",
+      "ipam": {
+        "type": "whereabouts",
+        "range": "172.23.0.0/24",
+        "range_start": "172.23.0.30",
+        "range_end": "172.23.0.70",
+        "routes": [
+           {
+             "dst": "172.24.0.0/16",
+             "gw" : "172.23.0.150"
+           }
+         ]
+      }
+    }
+EOF_CAT
+$ oc apply -n openstack -f octavia-nad.yaml
 ----
-$ oc patch openstackcontrolplane openstack --type=merge --patch '
+
+. Enable the {loadbalancer_service} service in {OpenShiftShort}:
++
+----
+$ oc patch openstackcontrolplane openstack -n openstack --type=merge --patch '
 spec:
+  ovn:
+    template:
+      ovnController:
+        networkAttachment: tenant
+        nicMappings:
+          octavia: octbr
   octavia:
     enabled: true
-    template: {}
+    template:
+      apacheContainerImage: registry.redhat.io/rhel8/httpd-24:latest
+      amphoraImageContainerImage: quay.io/gthiemonge/octavia-amphora-image
+      databaseInstance: openstack
+      octaviaHousekeeping:
+        networkAttachments:
+          - octavia
+      octaviaHealthManager:
+        networkAttachments:
+          - octavia
+      octaviaWorker:
+        networkAttachments:
+          - octavia
 '
 ----
+
+. Wait for the {loadbalancer_service} control plane services CRs to be ready:
++
+----
+$ oc wait --for condition=Ready --timeout=600s octavia.octavia.openstack.org/octavia
+----
+
+. Ensure that the {loadbalancer_service} is registered in the {identity_service}:
++
+----
+$ openstack service list | grep load-balancer
+| bd078ca6f90c4b86a48801f45eb6f0d7 | octavia   | load-balancer |
+$ openstack endpoint list | grep load-balancer
+| f1ae7756b6164baf9cb82a1a670067a2 | regionOne | octavia      | load-balancer | True    | public    | https://octavia-public-openstack.apps-crc.testing                     |
+| ff3222b4621843669e89843395213049 | regionOne | octavia      | load-balancer | True    | internal  | http://octavia-internal.openstack.svc:9876                            |
+----
+
+
+. Change the `ONBOOT` option in the network script for the management interface to `no`:
+
+.. Connect to each of the Controller nodes on the old control plane, for example, `overcloud-controller-0`.
+
+.. Open the configuration script for the management interface. For example:
++
+----
+$ sudo vi /etc/sysconfig/network-scripts/ifcfg-o-hm0
+----
++
+The exact name of the script depends on the previous `OctaviaMgmtPortDevName` setting.
+
+.. Change `ONBOOT=yes` to `ONBOOT=no` so that the interface does not get enabled automatically on reboot:
```

Comment:
"Change `ONBOOT=yes` to `ONBOOT=no` to ensure that the interface is disabled on reboot:"